examples/lambda-api-gateway/serverless.yml

```diff
@@ -0,0 +1,17 @@
+service: serverless-example
+
+plugins:
+  - serverless-plugin-typescript
+  - serverless-offline
+
+provider:
+  name: aws
+  runtime: nodejs14.x
+
+functions:
+  hello:
+    handler: server.lambdaHandler
+    events:
+      - http:
+          path: /greet
```

Comment:
I think for a more realistic example we should use `/{proxy+}`, otherwise this example will only work for the single procedure defined in code.

Reply:
Don't think there is a "correct" config here as they both have their advantages. This is an example folder so it represents a way of doing it.

Reply:
@KATT may correct me here, but I think if the client makes a batch request (and this is the default behaviour for the official clients), the path will be something like `/greet,greet?batch=2&input=...` (on mobile so I could be way off here) - so this handler wouldn't be invoked.

Reply:
@mmkal is correct

I added docs on this the other day https://trpc.io/docs/rpc